website/project/model.py

```diff
@@ -3308,6 +3297,16 @@ def get_admin_contributors_recursive(self, unique_users=False, *args, **kwargs):
                     else:
                         yield (contrib, node)
 
+    def get_admin_contributors(self, users):
```

Comment:
I need it because manage_contributor uses a slightly different contributor set when calling the function.